pykokkos/core/translators/bindings.py

```diff
@@ -124,12 +124,14 @@ def get_device_views(members: PyKokkosMembers) -> Dict[str, str]:
     return {v.declname: f"pk_d_{v.declname}" for v in members.views \
             if members.views[v] is not None}
 
-def generate_functor_instance(functor: str, members: PyKokkosMembers) -> str:
+def generate_functor_instance(functor: str, members: PyKokkosMembers, with_random_args: bool=True, functorExecSpace: Optional[str] = None) -> str:
     """
     Generate the functor instance
 
     :param functor: the name of the functor
     :param members: an object containing the fields and views
+    :param with_random_args: bool indicating if the constructor call should have the random args. default = True
+    :param functorExecSpace: optional parameter of type str that contains the ExecSpace template argument to the functor
```

Comment:
should be `functor_exec_space` to be consistent with our style

Reply:
done in [75997c](https://github.com/kokkos/pykokkos/pull/107/commits/75997c5fb6fb490f52b97c33f92592f0d758ca11)